test/specs/lib/felaSanitizeCssPlugin-test.ts

```diff
@@ -62,4 +62,13 @@ describe('felaSanitizeCssPlugin', () => {
 
     assertCssPropertyValue(`url('../../lib')`, true)
   })
+
+  describe('should pass arrays', () => {
```

Comment:
a bit confused with the test name, as it doesn't provide some necessary insights to me. Specifically, here:`should pass array` - pass untouched? Process somehow? If process, then what is the logic that will be applied?

Would expect tests to cover the following logic:
- for each item of an array CSS styles sanitization should be applied